src/foremast/utils/awslambda.py

```diff
@@ -99,22 +100,59 @@ def add_lambda_permissions(function='',
     session = boto3.Session(profile_name=env, region_name=region)
     lambda_client = session.client('lambda')
     response_action = None
+    prefixed_sid = 'foremast-' + statement_id
```

Comment:
Do you want it to be global to the package or to all of Foremast? Seems like if I make it global to all of Foremast I am over engineering this solution.